vm/src/stdlib/sys.rs

```diff
@@ -442,6 +442,69 @@ mod sys {
         .into_struct_sequence(vm))
     }
 
+    #[pyfunction]
+    fn __unraisablehook__(
+        unraisable: PyRef<UnraisableHookArgs>,
```

Comment:
e.g. for this (instead of the PyValue thing) you could write a custom TryFromObject for `UnraisableHookArgs` that just checks if `x.isinstance(Self::static_type())` and then extracts the tuple from there.